tsdb/ooo_head.go

```diff
@@ -135,13 +137,13 @@ func (oh *OOORangeHead) Tombstones() (tombstones.Reader, error) {
 	return tombstones.NewMemTombstones(), nil
 }
 
+var oooRangeHeadULID = ulid.MustParse("0000000000XXXX000RANGEHEAD")
```

Comment:
I'd suggest `0000000000XXXXXXXXX000HEAD` if you want to make it consistent with the in-order head's implementation. The in order head also has a RangeHead implementation that reuses the value you are setting above https://github.com/prometheus/prometheus/blob/f5913266a13429eef7507a024fe171118a2e673e/tsdb/head.go#L1410-L1419 

It would be setting the

Reply:
Hum, maybe I should set a different one for `RangeHead` instead?

Reply:
That works too 👍

Reply:
Pushed https://github.com/prometheus/prometheus/pull/13100/commits/3d7fdcf110f8087c39516532eaace792714cb828